redis/5.2/modules/ROOT/pages/redis-connector-reference.adoc

```diff
@@ -37,7 +37,8 @@ You can set configurations in Expression mode, except when Redis Connector is us
 ** HTTPS
 ** LDAPS
 ** DISABLED | The endpoint identification algorithm used by clients to validate the server hostname. The default value is an empty string, which means that this functionality is disabled. Clients, including client connections created by the broker for inter-broker communication, verify that the broker hostname matches the hostname in the broker's certificate. |  DISABLED |
-| TLS Configuration a| <<Tls>> |  Defines a TLS configuration that you can use from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the tls:context child element of the listener-config. The default is to use HTTP. | |
+| TLS Configuration a| <<Tls>> |  Configures TLS. If you are running the Mule app securely by using the HTTPS protocol, you must configure TLS in the connector.
```

Comment:
In the Connector Field Descriptions spreadsheet, the description for TLS is this:

Configures TLS. If using the HTTPS protocol, you must configure TLS.